pkg/cmd/admin/router/router.go

```diff
@@ -325,6 +325,9 @@ func generateSecretsConfig(cfg *RouterConfig, kClient *kclient.Client,
 		mounts = append(mounts, mount)
 	}
 
+	// default cert (which is a pem) is provided.
```

Comment:
If we were passed a default certificate argument (in pem form), check it, and make it into a secret